app/submitter/convert_payload_0_0_3.py

```diff
@@ -83,25 +93,30 @@ def add_list_collector_answers(
     )
     list_name = list_collector_block["for_list"]
     list_item_ids = list_store[list_name].items
-
-    for list_item_id in list_item_ids:
-        for answer_id in answers_ids_in_add_block:
-            answer = answer_store.get_answer(answer_id, list_item_id)
-            if answer:
-                answers_payload.add_or_update(answer)
+    if answers_ids_in_add_block:
```

Comment:
Ideally this should return early instead of nesting, but not a blocker.

Reply:
Yeah, I tried that and got coverage error, so would have to write bunch of tests for this if we return early so I just did the nesting as it's less intrusive.